submissions/Conversee12/Tiny JS World/index.js

```diff
@@ -0,0 +1,49 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Conversee12/a-tiny-JS-world.git
+   Web app: https://conversee12.github.io/a-tiny-JS-world/
+   */
+
+const Man = {
```

Comment:
Usually, names that start with a capital letter describe either class or constructor function. Regular objects (like here) are named with a lowercase first letter.